setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+VERSION = '1.0'
+
+from io import open
+from os import path
+from setuptools import setup
+
+here = path.abspath(path.dirname(__file__))
+
+
+def main():
+
+    packages = ['scripts', 'tests', 'images']
```

Comment:
Is there a reason to include the `test` and `images` as a sub package of the `DAGMC-viz` package ?

Reply:
You are right, there isn't really a reason to include `test`! However, the `images` folder is necessary because the CNERG logo is used by `data_loading.py`.